tests/test_commands.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
```

Comment:
Dunno why it was all started with different encoding header than the one we use in other places, but well, not that important ;)